src/containers/FilterBuilder.js

```diff
@@ -1,17 +1,106 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+
+import { fetchPopulation, savePopulation, getInstaCount } from '../actions/population';
 
 import PageHeader from '../components/Header/PageHeader';
+import FilterBuilderComponent from '../components/FilterBuilder/FilterBuilderComponent';
+
+class FilterBuilder extends Component {
+
+  constructor(...args) {
+    super(...args);
+    if(this.props.params.population_id){
+      this.state = this.setStateForEdit(this.props);
+    }
+    else {
+      this.state = this.setStateForCreate();
+    }
+
+  }
+
+  setStateForEdit(props) {
```

Comment:
Would it make more sense to have the dummy data in the actions? Then you can make sure the actions are working correctly.